From ee13d63db434b758f794f3e01ad298d955ce8a8a Mon Sep 17 00:00:00 2001
From: Paulo Vitor Magacho da Silva <pvmagacho@gmail.com>
Date: Sat, 29 Nov 2014 02:11:14 -0200
Subject: [PATCH] Fix package name for expandMedia and loadPlaylist native
 methods.

---
 libvlc/jni/libvlcjni-medialist.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libvlc/jni/libvlcjni-medialist.c b/libvlc/jni/libvlcjni-medialist.c
index 8b22b3e..f5a548f 100644
--- a/libvlc/jni/libvlcjni-medialist.c
+++ b/libvlc/jni/libvlcjni-medialist.c
@@ -75,14 +75,14 @@ static int expand_media_internal(JNIEnv *env, libvlc_instance_t* p_instance, job
     }
 }
 
-jint Java_org_videolan_libvlc_expandMedia(JNIEnv *env, jobject thiz, jint position, jobject children) {
+jint Java_org_videolan_libvlc_LibVLC_expandMedia(JNIEnv *env, jobject thiz, jint position, jobject children) {
     return (jint)expand_media_internal(env,
         getLibVlcInstance(env, thiz),
         children,
         (libvlc_media_t*)libvlc_media_player_get_media(getMediaPlayer(env, thiz)));
 }
 
-void Java_org_videolan_libvlc_loadPlaylist(JNIEnv *env, jobject thiz, jstring mrl, jobject items) {
+void Java_org_videolan_libvlc_LibVLC_loadPlaylist(JNIEnv *env, jobject thiz, jstring mrl, jobject items) {
     const char* p_mrl = (*env)->GetStringUTFChars(env, mrl, NULL);
     libvlc_instance_t *p_instance = getLibVlcInstance(env, thiz);
 
-- 
1.9.3 (Apple Git-50)

